platform/view/services/storage/driver/common/base.go

```diff
@@ -50,7 +49,7 @@ func (db *BaseDB[T]) BeginUpdate() error {
 	tx, err := db.newTransaction()
 	if err != nil {
 		db.txLock.Unlock()
-		return errors2.Wrapf(err, "error starting db transaction")
+		return errors.Wrapf(err, "error starting db transaction")
```

Comment:
While this was the wrong use of `errors.Wrapf` before .... Do you wanna use `errors.Wrap` here instead (and many other occurrences)?